shared/haml/user_header.haml

```diff
@@ -16,7 +16,7 @@
     .create_options{style: 'display: none'}
       - create_drop_down_items.each do |entry|
         %a.project_link_box{href: entry[:url]}
-          %img{src: entry[:image]}
+          %img{src: "/shared/images/fill-70x70/courses/#{entry[:image]}"}
```

Comment:
I was initially thinking we could do this with a `.map` in `create_header.rb` but this also works.